baseplate/lib/edge_context.py

```diff
@@ -167,7 +167,7 @@ class User(NamedTuple):
     """Wrapper for the user values in AuthenticationToken and the LoId cookie."""
 
     authentication_token: AuthenticationToken
-    loid: str
+    loid_: str
```

Comment:
```suggestion
    _loid: str
```

Reply:
It's not allowed in NamedTuple.

Reply:
:til:

Reply:
it's mypy complaining, actually: https://cloud.drone.io/reddit/baseplate.py/904/1/2

not sure if there's a way to override that behavior in mypy.

Reply:
https://docs.python.org/3/library/collections.html#collections.namedtuple

> Any valid Python identifier may be used for a fieldname except for names starting with an underscore.

so mypy's just enforcing the docs for namedtuple